storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst

```diff
@@ -83,39 +86,38 @@ server's :term:`datadir`.
 The database user needs the following privileges on the tables/databases to be
 backed up:
```

Comment:
BACKUP_ADMIN privilege is needed to query performance_schema.log_status, for LOCK INSTANCE FOR BACKUP and for LOCK TABLES FOR BACKUP.